docs/src/tutorials/temporal-properties.md

```diff
@@ -0,0 +1,416 @@
+# Temporal properties and counterexamples
+
+**Difficulty: Red trail – Medium**
+
+In this tutorial, we will show how Apalache can be used to decide temporal
+properties that are more general than invariants.
+This tutorial will be most useful to you if you have a basic understanding of
+linear temporal logic, e.g. the semantics of `<>` and `[]` operators.
+Further, we assume you are familiar with TLA+, but expert knowledge is not necessary.
```

Comment:
I wanted to give a link to a video lecture by Leslie Lamport on that topic, but it looks like there is no lecture where he is presenting `<>`, or maybe I have not found it. Maybe give a reference to "Specifying Systems"?

Reply:
There is a brief explanation of `<>` in video 9a.
But maybe Hillel's page on temporal operators is more informative? https://learntla.com/temporal-logic/operators/

Reply:
Added a link to Hilel's page